packages/loader/container-loader/src/container.ts

```diff
@@ -903,8 +903,8 @@ export class Container
 			document !== null &&
 			typeof document.addEventListener === "function" &&
 			document.addEventListener !== null;
-		// keep track of last time page was visible for telemetry
-		if (isDomAvailable) {
+		// keep track of last time page was visible for telemetry (on interactive clients only)
+		if (isDomAvailable && interactive) {
```

Comment:
I personally don't know this code that well, but my understanding is that the loader changes the slowest, which likely requires a backport fix. Where in the code do we create a second container? I know we create a second containerRuntime.

Reply:
That is actually a great question - @jatgarg  what  do you think ?